Source/Core/Core/HW/EXI/BBA/TAPServerConnection.cpp

```diff
@@ -0,0 +1,356 @@
+// Copyright 2020 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "Core/HW/EXI/EXI_DeviceEthernet.h"
+
+#ifdef _WIN32
+#include <winsock2.h>
+#include <ws2ipdef.h>
+#else
+#include <fcntl.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <sys/un.h>
+#include <unistd.h>
+#endif
+
+#include <cstdlib>
+#include <cstring>
+
+#include "Common/CommonFuncs.h"
+#include "Common/Logging/Log.h"
+#include "Common/StringUtil.h"
+#include "Core/HW/EXI/EXI_Device.h"
+
+namespace ExpansionInterface
+{
+
+#ifdef _WIN32
+using ws_ssize_t = int;
+#else
+#define closesocket close
+using ws_ssize_t = ssize_t;
+#endif
+
+#ifdef __LINUX__
```

Comment:
This should be `__linux__` (in lowercase) instead. It should fix the debian sigpipe crash.